husky-communication/husky-valueset-gen/src/main/java/org/projecthusky/valueset/utils/VsUtils.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * 
+ */
+package org.projecthusky.valueset.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.commons.io.IOUtils;
+
+/**
+ * 
+ */
+public class VsUtils {
+
+	/**
+	 * Method to download as a string
+	 * 
+	 * @param url
+	 *            the url to download form
+	 * @return the string of the downloaded content
+	 * @throws IOException
+	 *             occures on errors during download
+	 */
+	public static String downloadAsString(URL url) throws IOException {
+		URLConnection connection = url.openConnection();
+		connection.setRequestProperty("User-Agent", "Mozilla/5.0 Gecko/20100316 Firefox/3.6.2");
```

Comment:
In [the code generator](https://github.com/project-husky/code-generator/commit/1536bb92d23e20c9ad94db1e7c89e1cc7ffa5bbe), I used `Husky/1.0` to be a good net citizen.